test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/PravegaSegmentStoreK8sService.java

```diff
@@ -95,7 +95,7 @@ public boolean isRunning() {
                            int currentSegmentStoreCount = ((Double) pravegaSpec.get("segmentStoreReplicas")).intValue();
                            int currentBookkeeperCount = ((Double) bookkeeperSpec.get("replicas")).intValue();
                            log.debug("Current instance counts : Bookkeeper {} Controller {} SegmentStore {}.", currentBookkeeperCount,
-                                     currentControllerCount, currentSegmentStoreCount);
+                                    currentControllerCount, currentSegmentStoreCount);
```

Comment:
Same here.

Reply:
This changes was reverted same as pravega master release as this is not related to PR, only "one space" was removed which I update